modules/nf-core/cutesv/main.nf

```diff
@@ -0,0 +1,40 @@
+process CUTESV {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::cutesv=2.0.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/cutesv:1.0.12--pyhdfd78af_0' :
+        'quay.io/biocontainers/cutesv:2.0.2--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(bam), path(bai)
+    tuple val(meta2), path(fasta)
+
+    output:
+    tuple val(meta), path("*.vcf"), emit: vcf
+    path "versions.yml"                  , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def genotyping = params.enable_genotyping ? "--genotyping" : ''
+    """
+    cuteSV \
+        ${bam} \\
+        ${fasta} \\
+        ${prefix}.vcf \\
```

Comment:
Does this tool output bgzipped output? If so, having bgzipped output is preffered

Reply:
for outputting a `.bgzip` file would require an additional command
```
bgzip -c tools/cutesv/cutesv.vcf > tools/cutesv/cutesv.vcf.gz
```

Reply:
That would be perfect (if `bgzip` is in the containers for CuteSV, otherwise it's fine to leave it unzipped)

Reply:
`bgzip` is not in the container

Reply:
Ahh too bad, it's fine then :)

Reply:
Hi @nvnieuwk, can you approve the PR, please?